### ISSUE TITLE
Error during stringtable import.

### ISSUE THREAD

**Issue by StickieBE** [kayler-renslow/arma-dialog-creator#37]
### Version and Build
v: 1.06 / b: 54 / Java: 9.04

### Summary
Error when trying to import a default stringtable.xml.
**stringtable.xml:**
```
// stringtable.xml
<?xml version="1.0" encoding="utf-8" ?>
<Project name="Any Name">
	<Package name="Mission One">
		<Container name="Some Words">
			<Key ID="STR_myKey">
				<Original>Hey there</Original>
			</Key>
		</Container>
	</Package>
</Project>
```

### Detail
**Screenshot:** 
http://prntscr.com/kf6yn7

**Stack Trace:**
```
java.lang.NoSuchMethodError: org.fxmisc.richtext.TextExt.impl_selectionFillProperty()Ljavafx/beans/property/ObjectProperty;
	at org.fxmisc.richtext.ParagraphText.<init>(ParagraphText.java:110)
	at org.fxmisc.richtext.ParagraphBox.<init>(ParagraphBox.java:75)
	at org.fxmisc.richtext.StyledTextArea.createCell(StyledTextArea.java:1120)
	at org.fxmisc.richtext.StyledTextArea.lambda$new$3(StyledTextArea.java:585)
	at org.fxmisc.flowless.CellPool.getCell(CellPool.java:20)
	at org.fxmisc.flowless.CellListManager.cellForItem(CellListManager.java:75)
	at org.reactfx.collection.MappedList.get(MappedList.java:27)
	at org.reactfx.collection.MemoizationListImpl.get(MemoizationList.java:99)
	at org.fxmisc.flowless.CellListManager.getCell(CellListManager.java:64)
	at org.fxmisc.flowless.CellPositioner.getSizedCell(CellPositioner.java:129)
	at org.fxmisc.flowless.CellPositioner.placeStartAt(CellPositioner.java:95)
	at org.fxmisc.flowless.Navigator.placeStartAtMayCrop(Navigator.java:183)
	at org.fxmisc.flowless.Navigator.visit(Navigator.java:111)
	at org.fxmisc.flowless.StartOffStart.accept(TargetPosition.java:49)
	at org.fxmisc.flowless.Navigator.layoutChildren(Navigator.java:67)
	at javafx.graphics/javafx.scene.Parent.layout(Unknown Source)
	at org.fxmisc.flowless.VirtualFlow.layoutChildren(VirtualFlow.java:165)
	at javafx.graphics/javafx.scene.Parent.layout(Unknown Source)
	at javafx.graphics/javafx.scene.Parent.layout(Unknown Source)
	at javafx.graphics/javafx.scene.Parent.layout(Unknown Source)
	at javafx.graphics/javafx.scene.Parent.layout(Unknown Source)
	at javafx.graphics/javafx.scene.Parent.layout(Unknown Source)
	at javafx.graphics/javafx.scene.Parent.layout(Unknown Source)
	at javafx.graphics/javafx.scene.Parent.layout(Unknown Source)
	at javafx.graphics/javafx.scene.Parent.layout(Unknown Source)
	at javafx.graphics/javafx.scene.Scene.doLayoutPass(Unknown Source)
	at javafx.graphics/javafx.scene.Scene.preferredSize(Unknown Source)
	at javafx.graphics/javafx.scene.Scene$2.preferredSize(Unknown Source)
	at javafx.graphics/com.sun.javafx.scene.SceneHelper.preferredSize(Unknown Source)
	at javafx.graphics/javafx.stage.Window$12.invalidated(Unknown Source)
	at javafx.base/javafx.beans.property.BooleanPropertyBase.markInvalid(Unknown Source)
	at javafx.base/javafx.beans.property.BooleanPropertyBase.set(Unknown Source)
	at javafx.graphics/javafx.stage.Window.setShowing(Unknown Source)
	at javafx.graphics/javafx.stage.Window.show(Unknown Source)
	at javafx.graphics/javafx.stage.Stage.show(Unknown Source)
	at com.kaylerrenslow.armaDialogCreator.gui.popup.StagePopup.show(StagePopup.java:154)
	at com.kaylerrenslow.armaDialogCreator.gui.main.actions.mainMenu.EditStringTableAction.handle(EditStringTableAction.java:48)
	at com.kaylerrenslow.armaDialogCreator.gui.main.actions.mainMenu.EditStringTableAction.handle(EditStringTableAction.java:24)
	at javafx.base/com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(Unknown Source)
	at javafx.base/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at javafx.base/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at javafx.base/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at javafx.base/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at javafx.base/com.sun.javafx.event.EventUtil.fireEventImpl(Unknown Source)
	at javafx.base/com.sun.javafx.event.EventUtil.fireEvent(Unknown Source)
	at javafx.base/javafx.event.Event.fireEvent(Unknown Source)
	at javafx.controls/javafx.scene.control.MenuItem.fire(Unknown Source)
	at javafx.controls/com.sun.javafx.scene.control.ContextMenuContent$MenuItemContainer.doSelect(Unknown Source)
	at javafx.controls/com.sun.javafx.scene.control.ContextMenuContent$MenuItemContainer.lambda$createChildren$12(Unknown Source)
	at javafx.base/com.sun.javafx.event.CompositeEventHandler$NormalEventHandlerRecord.handleBubblingEvent(Unknown Source)
	at javafx.base/com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(Unknown Source)
	at javafx.base/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at javafx.base/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at javafx.base/com.sun.javafx.event.CompositeEventDispatcher.dispatchBubblingEvent(Unknown Source)
	at javafx.base/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at javafx.base/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at javafx.base/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at javafx.base/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at javafx.base/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at javafx.base/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at javafx.base/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at javafx.base/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at javafx.base/com.sun.javafx.event.EventUtil.fireEventImpl(Unknown Source)
	at javafx.base/com.sun.javafx.event.EventUtil.fireEvent(Unknown Source)
	at javafx.base/javafx.event.Event.fireEvent(Unknown Source)
	at javafx.graphics/javafx.scene.Scene$MouseHandler.process(Unknown Source)
	at javafx.graphics/javafx.scene.Scene$MouseHandler.access$1300(Unknown Source)
	at javafx.graphics/javafx.scene.Scene.processMouseEvent(Unknown Source)
	at javafx.graphics/javafx.scene.Scene$ScenePeerListener.mouseEvent(Unknown Source)
	at javafx.graphics/com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(Unknown Source)
	at javafx.graphics/com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at javafx.graphics/com.sun.javafx.tk.quantum.GlassViewEventHandler.lambda$handleMouseEvent$2(Unknown Source)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.runWithoutRenderLock(Unknown Source)
	at javafx.graphics/com.sun.javafx.tk.quantum.GlassViewEventHandler.handleMouseEvent(Unknown Source)
	at javafx.graphics/com.sun.glass.ui.View.handleMouseEvent(Unknown Source)
	at javafx.graphics/com.sun.glass.ui.View.notifyMouse(Unknown Source)
	at javafx.graphics/com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at javafx.graphics/com.sun.glass.ui.win.WinApplication.lambda$runLoop$3(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)

THREAD:Arma Dialog Creator JavaFX Thread
```

Thanks again for ADC!

**Comment by kayler-renslow** [comment]
This appears to be a bug with the GUI library rather than the StringTable or StringTable importer itself. Doesn't look like there's much I can do besides just update the library.

**Comment by kayler-renslow** [comment]
I'm presuming this is fixed in 1.0.7 since I updated the libs. If it isn't fixed, re-open the issue.

**Comment by StickieBE** [comment]
Re-opening issue because bug is still present.

Stacktrace:
`java.lang.IllegalArgumentException: id 'STR_myKey' isn't proper
	at com.kaylerrenslow.armaDialogCreator.arma.stringtable.StringTableKey.setKey(StringTableKey.java:68)
	at com.kaylerrenslow.armaDialogCreator.arma.stringtable.impl.StringTableKeyImpl.<init>(StringTableKeyImpl.java:40)
	at com.kaylerrenslow.armaDialogCreator.data.xml.DefaultStringTableXmlParser.fetchKeys(DefaultStringTableXmlParser.java:93)
	at com.kaylerrenslow.armaDialogCreator.data.xml.DefaultStringTableXmlParser.loadContainerElements(DefaultStringTableXmlParser.java:67)
	at com.kaylerrenslow.armaDialogCreator.data.xml.DefaultStringTableXmlParser.loadPackageElements(DefaultStringTableXmlParser.java:57)
	at com.kaylerrenslow.armaDialogCreator.data.xml.DefaultStringTableXmlParser.createStringTableInstance(DefaultStringTableXmlParser.java:44)
	at com.kaylerrenslow.armaDialogCreator.gui.main.actions.mainMenu.EditStringTableAction.handle(EditStringTableAction.java:39)
	at com.kaylerrenslow.armaDialogCreator.gui.main.actions.mainMenu.EditStringTableAction.handle(EditStringTableAction.java:24)
	at com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(CompositeEventHandler.java:86)
	at com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:238)
	at com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:191)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:58)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at com.sun.javafx.event.EventUtil.fireEventImpl(EventUtil.java:74)
	at com.sun.javafx.event.EventUtil.fireEvent(EventUtil.java:49)
	at javafx.event.Event.fireEvent(Event.java:198)
	at javafx.scene.control.MenuItem.fire(MenuItem.java:462)
	at com.sun.javafx.scene.control.skin.ContextMenuContent$MenuItemContainer.doSelect(ContextMenuContent.java:1405)
	at com.sun.javafx.scene.control.skin.ContextMenuContent$MenuItemContainer.lambda$createChildren$343(ContextMenuContent.java:1358)
	at com.sun.javafx.event.CompositeEventHandler$NormalEventHandlerRecord.handleBubblingEvent(CompositeEventHandler.java:218)
	at com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(CompositeEventHandler.java:80)
	at com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:238)
	at com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:191)
	at com.sun.javafx.event.CompositeEventDispatcher.dispatchBubblingEvent(CompositeEventDispatcher.java:59)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:58)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at com.sun.javafx.event.EventUtil.fireEventImpl(EventUtil.java:74)
	at com.sun.javafx.event.EventUtil.fireEvent(EventUtil.java:54)
	at javafx.event.Event.fireEvent(Event.java:198)
	at javafx.scene.Scene$MouseHandler.process(Scene.java:3757)
	at javafx.scene.Scene$MouseHandler.access$1500(Scene.java:3485)
	at javafx.scene.Scene.impl_processMouseEvent(Scene.java:1762)
	at javafx.scene.Scene$ScenePeerListener.mouseEvent(Scene.java:2494)
	at com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(GlassViewEventHandler.java:394)
	at com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(GlassViewEventHandler.java:295)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.tk.quantum.GlassViewEventHandler.lambda$handleMouseEvent$353(GlassViewEventHandler.java:432)
	at com.sun.javafx.tk.quantum.QuantumToolkit.runWithoutRenderLock(QuantumToolkit.java:389)
	at com.sun.javafx.tk.quantum.GlassViewEventHandler.handleMouseEvent(GlassViewEventHandler.java:431)
	at com.sun.glass.ui.View.handleMouseEvent(View.java:555)
	at com.sun.glass.ui.View.notifyMouse(View.java:937)
	at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at com.sun.glass.ui.win.WinApplication.lambda$null$147(WinApplication.java:177)
	at java.lang.Thread.run(Unknown Source)

THREAD:Arma Dialog Creator JavaFX Thread`
